### PR TITLE
9.0 from 8.0

### PR DESCRIPTION
**Pull request by lmignon**
Migration to version 9.0
- [x] connector (@lmignon)
  - [x] forward port of latest changes from 8.0 to 9.0
  - [x] make the module installable (@lmignon)
  - [x] remove the old workers code (@lmignon)
  - [x] integrate PR #165 (@lmignon)

### CODE REVIEW

**Comment by lmignon** [comment]
Work in progress since I have not yet made functional tests.

File: connector/__openerp__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ##############################################################################
 
 {'name': 'Connector',
- 'version': '8.0.3.2.0',
+ 'version': '9.0.0.1.0',
```
Comment (sbidoul) [inline connector/__openerp__.py]: 9.0.1.0.0

**Comment by lmignon** [comment]
Removed `ẁork in progress`. Successfully tested with https://github.com/OCA/connector-cmis/pull/10

**Comment by yvaucher** [comment]
@lmignon For the docs failure you can probably do the same as here:

https://github.com/OCA/geospatial/commit/9baa5b799648b9302d7ca6a0f8d9d7fe6cd4e180

**Comment by lmignon** [comment]
Thank you @yvaucher. Done

**Comment by guewen** [comment]
Excellent! thanks @lmignon 
Do you think we should remove the deprecated functions? (grep for 'log_deprecated' and DeprecatedClass)

**Comment by parthivgls** [comment]
:+1:

**Comment by lmignon** [comment]
@guewen  You're right, API breakage should occurs only on major release. I'll remove the deprecated stuffs

**Comment by guewen** [comment]
I released 8.0.3.3.0 (https://github.com/OCA/connector/commit/646ea58f13ff29c9a5b6846e2fb6f4d8de032c09) so a rebase would be cool

**Comment by lmignon** [comment]
@guewen deprecated APIs removed and rebase done

**Comment by guewen** [comment]
thanks!

:+1:

**Comment by sbidoul** [comment]
:+1:

**Comment by SodexisTeam** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Great work!

Merging...

**Comment by sbidoul** [comment]
Merging too... :wink:

**Comment by pedrobaeza** [comment]
Oh, I thought I clicked the button...

**Comment by bistaray** [comment]
Is this the "Odoo connector" mentioned at http://odoo-connector.com/ for which a crowd funding campaign was run?  https://www.indiegogo.com/projects/odoo-connector-odoo-magento-connector-for-odoo-9#/

I don't need the Magento part, I just need the "Odoo connector framework" for v9.

**Comment by lasley** [comment]
Yes & it works like a charm.

**Pull request merged**